crates/cairo-profiler/src/trace_reader.rs

```diff
@@ -78,22 +78,35 @@ impl Sample {
 }
 
 pub struct EntryPointId {
-    address: String,
-    selector: String,
-    contract_name: Option<String>,
-    function_name: Option<String>,
+    address: Option<String>,
+    selector: Option<String>,
+    contract_name: String,
+    function_name: String,
```

Comment:
We shouldn't change this struct, we should still require address and selector and make contract name and function name optional. A good example of that are fork tests for forge, which won't generate contract names and function names.

Reply:
Add a comment explaining the logic pls